index.js

```diff
@@ -135,30 +179,33 @@ function nowServing(line) {
   return `Currently serving ${line.shift()}.`
 }
 
-const line = []
+const line = [0]
 
 // ***** Scope & Closures - Question 1 *****
-
-
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
 
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 1`
+function takeATicketNumber(line) {
+    line.push(line[line.length - 1] + 1)
+    console.log(`Welcome. You are ticket number ${line.length - 1}`)
+}
```

Comment:
Nice solution. 

The section was hoping to practice a little with scope. The intended answer would create a counter variable in the global scope which the function would then use to add people/numbers to the line.